Test failure in ydb/tests/compatibility/result_set_format/test_result_set_arrow.py.TestResultSetArrow.test_column_alias[restart_prestable-26-1_to_prestable-26-1-COLUMN]

ydb/tests/compatibility/result_set_format/test_result_set_arrow.py.TestResultSetArrow.test_column_alias[restart_prestable-26-1_to_prestable-26-1-COLUMN]

**Test Failure**

**Build Information:**
- Version: 8ea7ec920e76164ab06fa8c5819d5daeea7508ba
- Build Preset: relwithdebinfo
- Branch: main

**Test Report:** [View report](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21044181712/ya-main-x86-64/try_1/ya-test.html#FAIL)

**Run Command:**
```
./ya make -ttt --build "relwithdebinfo" -F 'test_result_set_arrow.py::TestResultSetArrow::test_column_alias[restart_prestable-26-1_to_prestable-26-1-COLUMN]' ydb/tests/compatibility/result_set_format
```

**Owner:** [TEAM:@ydb-platform/engineering](https://github.com/orgs/ydb-platform/teams/engineering)

**Success rate (last 4 days):**
MAIN: 0% (0/27)

**Test run history:** [link](https://datalens.yandex/34xnbsom67hcq?branch=main&full_name=ydb/tests/compatibility/result_set_format/test_result_set_arrow.py.TestResultSetArrow.test_column_alias[restart_prestable-26-1_to_prestable-26-1-COLUMN]&build_type=relwithdebinfo)

More info in [dashboard](https://datalens.yandex/4un3zdm0zcnyr)

**Logs:**
- [log](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21044181712/ya-main-x86-64/try_1/artifacts/logs/ydb/tests/compatibility/result_set_format/test-results/py3test/testing_out_stuff/test_result_set_arrow.py.TestResultSetArrow.test_column_alias.restart_prestable-26-1_to_prestable-26-1-COLUMN.log)
- [logsdir](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/21044181712/ya-main-x86-64/try_1/test_artifacts/ydb-tests-compatibility-result_set_format.zip)

**Error Summary:**
```
ydb/tests/compatibility/result_set_format/test_result_set_arrow.py:269: in _try_execute
    return pool.execute_with_retries(
contrib/python/ydb/py3/ydb/query/pool.py:252: in execute_with_retries
    return retry_operation_sync(wrapped_callee, retry_settings)
contrib/python/ydb/py3/ydb/retries.py:137: in retry_operation_sync
    for next_opt in opt_generator:
contrib/python/ydb/py3/ydb/retries.py:98: in retry_operation_impl
    result = YdbRetryOperationFinalResult(callee(*args, **kwargs))
contrib/python/ydb/py3/ydb/query/pool.py:250: in wrapped_callee
    return [result_set for result_set in it]
contrib/python/ydb/py3/ydb/_utilities.py:173: in __next__
    return self._next()
contrib/python/ydb/py3/ydb/_utilities.py:164: in _next
    res = self.wrapper(next(self.it))
contrib/python/ydb/py3/ydb/query/session.py:390: in <lambda>
    lambda resp: base.wrap_execute_query_response(
contrib/python/ydb/py3/ydb/query/base.py:211: in decorator
    return func(rpc_state, response_pb, session_state, *args, **kwargs)
contrib/python/ydb/py3/ydb/query/base.py:229: in wrap_execute_query_response
    issues._process_response(response_pb)
contrib/python/ydb/py3/ydb/issues.py:237: in _process_response
    raise exc_obj(_format_response(response_proto), response_proto.issues)
E   ydb.issues.SchemeError: message: "Executing ESchemeOpCreateColumnTable" issue_code: 2003 severity: 1 issues { message: "Type \'Bool\' specified for column \'col_Bool\', but support for bool is disabled (EnableColumnshardBool feature flag is off)" issue_code: 2003 severity: 1 } ,message: "Query invalidated on scheme/internal error during Scheme execution" issue_code: 2019 severity: 1 (server_code: 400070)

During handling of the above exception, another exception occurred:
ydb/tests/compatibility/result_set_format/test_result_set_arrow.py:242: in test_column_alias
    self._create_table(table_name, store_type)
ydb/tests/compatibility/result_set_format/test_result_set_arrow.py:290: in _create_table
    self._try_execute(query)
ydb/tests/compatibility/result_set_format/test_result_set_arrow.py:278: in _try_execute
    assert False, f"Failed query `{query}`, error: {e}"
E   AssertionError: Failed query `
E             CREATE TABLE `test_arrow` (
E                 pk_Uint64 Uint64 NOT NULL, col_Int64 Int64, col_Uint64 Uint64, col_Int32 Int32, col_Uint32 Uint32, col_Int16 Int16, col_Uint16 Uint16, col_Int8 Int8, col_Uint8 Uint8, col_Bool Bool, col_Decimal150 Decimal(15,0), col_Decimal229 Decimal(22,9), col_Decimal3510 Decimal(35,10), col_String String, col_Utf8 Utf8, col_Date Date, col_Datetime Datetime, col_Timestamp Timestamp, col_Date32 Date32, col_Datetime64 Datetime64, col_Timestamp64 Timestamp64, col_Interval64 Interval64, col_Float Float, col_Double Double, col_Json Json, col_JsonDocument JsonDocument, col_Yson Yson,
E                 PRIMARY KEY(
E                     pk_Uint64
E                     )
E                 
E                 )
E         
E                 WITH (
E                     STORE = COLUMN
E                 )
E             `, error: message: "Executing ESchemeOpCreateColumnTable" issue_code: 2003 severity: 1 issues { message: "Type \'Bool\' specified for column \'col_Bool\', but support for bool is disabled (EnableColumnshardBool feature flag is off)" issue_code: 2003 severity: 1 } ,message: "Query invalidated on scheme/internal error during Scheme execution" issue_code: 2019 severity: 1 (server_code: 400070)
E   assert False
```

--- a/ydb/tests/compatibility/result_set_format/test_result_set_arrow.py
+++ b/ydb/tests/compatibility/result_set_format/test_result_set_arrow.py
@@ -72,7 +72,7 @@ class TestResultSetArrow(RestartToAnotherVersionFixture):
         if min(self.versions) < (25, 3, 2):
             pytest.skip("Result set formats are not supported in <= 25.3.1")
 
-        if min(self.versions) < (26, 1):
+        if min(self.versions) < (26, 2):
             types_not_supported_yet_in_columnshard.add("Bool")
 
         supported_pk_types = pk_types if store_type == "ROW" else {k: v for k, v in pk_types.items() if k not in types_not_supported_yet_in_columnshard}

--- a/ydb/tests/compatibility/result_set_format/test_result_set_value.py
+++ b/ydb/tests/compatibility/result_set_format/test_result_set_value.py
@@ -38,7 +38,7 @@ class TestResultSetValue(RestartToAnotherVersionFixture):
         if min(self.versions) < (25, 3, 2):
             pytest.skip("Result set formats are not supported in <= 25.3.1")
 
-        if min(self.versions) < (26, 1):
+        if min(self.versions) < (26, 2):
             types_not_supported_yet_in_columnshard.add("Bool")
 
         supported_pk_types = pk_types if store_type == "ROW" else {k: v for k, v in pk_types.items() if k not in types_not_supported_yet_in_columnshard}

